### PR TITLE
Add some shields to README.md

### PR DESCRIPTION
**Pull request by garrison**
Preview at: https://github.com/IBM-Quantum-Prototypes/quantum-kernel-training/tree/shields#readme

The precise list of shields will surely evolve, but this seems like a reasonable starting point.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+<!-- SHIELDS -->
+[![Qiskit](https://img.shields.io/badge/Qiskit-0.34.1%20or%20later-informational)](https://qiskit.org/)
+[![Python](https://img.shields.io/badge/Python-3.9-informational)](https://www.python.org/)
+[![License](https://img.shields.io/github/license/IBM-Quantum-Prototypes/quantum-kernel-training?color=informational&label=License)](https://github.com/IBM-Quantum-Prototypes/quantum-kernel-training/blob/main/LICENSE.txt)
```
Comment (calebj15) [inline README.md:4]: Do we want to back-support Qiskit (1 version? 2?)

Comment (calebj15) [reply README.md:4]: No

**Review by calebj15 — CHANGES_REQUESTED** [review]
Ad OS shield {Linux(test for ubuntu and ubi), macOS, Windows}

**Comment by garrison** [comment]
How does the current preview look?

**Comment by garrison** [comment]
BTW, I didn't enable a coverage badge because there is not a coverage workflow in this repository -- everything is a notebook test.

**Review by calebj15 — APPROVED** [review]

**Pull request merged**